manifests/4.10/kubernetes-nmstate-operator.v4.10.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,233 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    operatorframework.io/initialization-resource: |-
+      {
+        "apiVersion": "nmstate.io/v1",
+        "kind": "NMState",
+        "metadata": {
+          "name": "nmstate"
+        },
+        "spec": {
+          "nodeSelector": {
+            "beta.kubernetes.io/arch": "amd64"
```

Comment:
I wonder if eventually we will need to remove this when we have support for other platforms. Should be fine for now though.